sky/utils/cli_utils/status_utils.py

```diff
@@ -45,6 +45,7 @@ def show_status_table(show_all: bool = False, refresh: bool = False):
                      _get_resources,
                      trunc_length=70 if not show_all else 0),
         StatusColumn('REGION', _get_region, show_by_default=False),
+        StatusColumn('REQUESTED_ZONE', _get_zone, show_by_default=False),
```

Comment:
`REQUESTED_ZONE` -> `ZONE`? As the launched cluster should always align with the requested zone.

Reply:
Changed!